eng/pipelines/mirror-repos.yml

```diff
@@ -11,61 +11,62 @@ jobs:
     displayName: Sync public repos
 
     steps:
+    - template: /eng/common/pipelines/templates/steps/set-default-branch.yml
```

Comment:
I don't think this will work. The repo mirroring pipeline will checkout the _azure-sdk-tools_ repository so the default branch that is set here would be that repo, not the repos that are being synced. I think that you'll need to adapt ```set-default-branch.yml``` to be able to be pointed at a particular path and then called within the ```sync-repo-branch.yml``` for each repo that is being synchronized.

Reply:
Have the setting default branch inside of `sync-repo-branch.yml`.